database/impl/src/main/kotlin/app/aaps/database/entities/GlucoseValue.kt

```diff
@@ -103,6 +103,7 @@ data class GlucoseValue(
         AIDEX,
         RANDOM,
         UNKNOWN,
+        SyaiTag,
```

Comment:
I think SYAITAG should be put in uppercase to be consistant with the other items of the list

Reply:
We have updated